project.php

```diff
@@ -155,27 +155,19 @@
 
 function do_update_pp_activity()
 {
-// If the project has been checked out for more than 90 days and
-// the pp-er loads the page, it's interpreted as a sign of
-// activity. Update the database accordingly.
+    // If the project is checked out and the PPer loads the page, it's
+    // interpreted as a sign of activity. Update the database accordingly.
 
     global $project;
 
     if ( $project->state != PROJ_POST_FIRST_CHECKED_OUT ) return;
 
     if (! $project->PPer_is_current_user) return;
 
-    // 7776000 = 90*24*60*60 seconds
-    if ($project->modifieddate < time()-7776000) {
-      // Modified more than 90 days ago. The PP-er,
-      // by loading this page, is reporting
-      // activity.
-
-      $projectid = $project->projectid;
-      $now = time();
-      mysqli_query(DPDatabase::get_connection(), "UPDATE projects SET modifieddate=$now WHERE projectid='$projectid'");
-      $project->modifieddate = $now;
-    }
+    $projectid = $project->projectid;
+    $now = time();
+    mysqli_query(DPDatabase::get_connection(), "UPDATE projects SET modifieddate=$now WHERE projectid='$projectid'");
```

Comment:
Maybe we should stop (ab)using the project's modifieddate for this purpose.

`user_project_info.t_latest_home_visit` already records each user's visit to a project's "home page", so we could just look that up for the PPer when we want to know whether to remind them.

Reply:
Good idea - that would be a better option. I will get a PR up for that.